src/content/keyAutoAdd/lib/keyAutoAdd.ts

```diff
@@ -62,8 +62,9 @@ export class KeyAutoAdd {
 
     if (size === 'notification') {
       styles = {
-        width: '22rem',
-        height: '8rem',
+        width: '22em',
+        height: '8em',
+        fontSize: '16px',
```

Comment:
GateHub has `fontSize: 10px` on root, so our notification UI had size reduced. This fixes it.